modules/microshift-storage-creating-vol-snapshot.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// microshift/volume-snapshots-microshift.adoc
+
+:_content-type: PROCEDURE
+[id="creating-a-volume-snapshotting_{context}"]
+= Creating a volume snapshot
+
+To create a snapshot of a {product-title} storage volume, you must first configure {op-system-ostree} and the cluster. In the following example procedure, the pod that the source volume is mounted to is deleted. Deleting the pod prevents data continuing to be written to it during snapshot creation. If the pod is managed with a replication controller, meaning that a new pod is instantly created when one is deleted, temporarily scale the replica count to zero instead of deleting the pod.
+
+[NOTE]
+====
+Volume snapshots are located on the same devices as the original data. To use the volume snapshots as backups, move the snapshots to a secure location.
+====
+
+.Prerequisites
+
+. User has root access to a {product-title} cluster.
+. A {product-title} cluster is running.
+. A device class defines an LVM thin-pool.
+. A `volumeSnapshotClass` specifies `driver: topolvm.io`.
+. Any workload attached to the source PVC is paused or deleted. This helps avoid data corruption.
+
+[IMPORTANT]
+====
+All writes to the volume must be halted while you are creating the snapshot. If you do not halt writes, your data might be corrupted.
+====
+
+.Procedure
+
+. Delete the pod to ensure that no data is written to the volume during snapshotting by running the following command:
++
+[source, terminal]
+----
+$ oc delete my-pod
+----
+
+. Optional: If the pod is managed with a replication controller, meaning that a new pod is instantly created when one is deleted, temporarily scale the replica count to zero instead of deleting the pod.
+
+. Once the pod has been deleted, run a command similar to the example that follows. Insert your own configuration details.
++
+.Example snapshot configuration
++
+[source, terminal]
+----
+# oc apply -f <<EOF
```

Comment:
We should scan the shell prompts in these procedures to make sure they're consistent. This one is for root, the next one is for a regular user. I don't think the user needs to be root for any of these steps, right @copejon ?

Reply:
Yes. And agreed - we should stick to non-root command examples.

Reply:
I made these all `$`. Let me know if we need to stick a sudo in them anywhere.